src/tv2-common/actions/executeAction.ts

```diff
@@ -1837,6 +1847,59 @@ function executeActionRecallLastDVE<
 	}
 }
 
+function executeActionPlayGraphics<
+	StudioConfig extends TV2StudioConfigBase,
+	ShowStyleConfig extends TV2BlueprintConfigBase<StudioConfig>
+>(
+	context: ITV2ActionExecutionContext,
+	settings: ActionExecutionSettings<StudioConfig, ShowStyleConfig>,
+	actionId: string,
+	_userData: ActionPlayGraphics
+) {
+	const internalGraphic: InternalGraphic = getInternalGraphic(
+		settings.getConfig(context),
+		_userData.graphic,
+		true,
+		undefined,
+		undefined,
+		undefined,
+		undefined
+	)
+	const pieces: IBlueprintPiece[] = []
+	if (IsTargetingOVL(_userData.graphic.target)) {
```

Comment:
You can flip the condition and just return instead of nesting. 
Also, you don't use "internalGraphic" unless you go inside your conditional, so you can move the fetching of the internal graphic inside your statement, so you don't fetch it unless you need it